bukkit/pom.xml

```diff
@@ -84,7 +83,7 @@
         <dependency>
             <groupId>me.lokka30</groupId>
             <artifactId>treasury-api-bukkit</artifactId>
-            <version>${project.version}</version>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
Would it be worth to have properties for each api module and use the properties on dependencies instead of putting just the version?

Reply:
couldn't understand your question ; do you mean having a property which holds the maven api version so it can be used in dependencies?